packages/tsdl_image/tsdl_image.0.3.0/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+maintainer: "Julian Squires <julian@cipht.net>"
+authors: ["Julian Squires <julian@cipht.net>"]
+homepage: "http://github.com/tokenrove/tsdl-image"
+dev-repo: "git+https://github.com/tokenrove/tsdl-image.git"
+bug-reports: "http://github.com/tokenrove/tsdl-image/issues"
+tags: [ "bindings" "graphics" ]
+license: "BSD-3-Clause"
+depends: [
+  "ocaml" {>= "4.02"}
+  "ctypes" {>= "0.4.0"}
+  "ctypes-foreign"
+  "tsdl" {>= "0.9.0"}
+  "result"
+  "dune" {build & >= "1.11.0"}
```

Comment:
```suggestion
  "dune" {>= "1.11.0"}
```
Dune shouldn't be tagged as a build dependency anymore (see #14266)

Reply:
This seems like a significant issue and I might cancel this transition as a consequence.

Reply:
I'm curious to know why it is an issue for you if you have time to tell?